Automattic/sensei
# Update placeholder for course outline block to include blank or AI option

Resolves #6956.

## Proposed Changes
Introduces a `window.sensei.aiCourseOutline` feature flag. If enabled, it shows the new UI:

![Screenshot 2023-06-14 at 12 03 56 PM](https://github.com/Automattic/sensei/assets/1190420/bc12e197-11b6-41a5-86bc-2bd22352c122)

Is disabled, it shows the old UI:

![Screenshot 2023-06-14 at 12 04 38 PM](https://github.com/Automattic/sensei/assets/1190420/c972ea6a-6041-49cd-9ced-2c924eb6b9b1)

## Testing Instructions

1. Run `npm run start` or `npm run build` for Sensei LMS.
2. Create a new course.
3. Step through the wizard and select the default layout.
4. Remove all lessons from the course outline in order to see the placeholder.
5. Check that the UI matches the [design](https://www.figma.com/file/NyhJy9d7SwvDoIAXecN7eH/Course-Creation?type=design&node-id=4352-33920&t=EGtFWFV4yaT8Efud-0).
6. Test it at different resolutions. At resolutions < 600px, the options should stack.
7. Set [this line](https://github.com/Automattic/sensei/compare/update/course-outline-placeholder-options?expand=1#diff-a37580803b282584a79424a39affab28b2d702ece20c9d4382d61cbb8c598b18R129) to `false` to disable the feature flag.
8. Refresh the browser and check that the old UI is visible.

## Pre-Merge Checklist
